vms/platformvm/state/state.go

```diff
@@ -420,6 +429,18 @@ type txAndStatus struct {
 	status status.Status
 }
 
+type wrappedFxOwner struct {
```

Comment:
I wanna name this something else... just not huge on using `wrapped*` as a name since it's a little hard for me to know what this is without looking at the code closely (I had to read the surrounding cache code to figure out what this was doing). Can we call this something like `subnetOwnerSize` or anything along the lines of `*ownerSize`?

Reply:
I made it `fxOwnerAndSize` to match `txAndStatus`. Don't like the stutter of `fxOwnerAndSizeSize()` tho :(

Reply:
Sounds good!

Reply:
Removed fxOwnerAndSizeSize since it is always `ids.IDLen + fxOwnerAndSize.size`